component/client/chat/ClientChat.tsx

```diff
@@ -21,6 +22,14 @@ class ClientChat extends React.Component<IClientChatProps, IClientChatState> {
   constructor(props: IClientChatProps) {
     super(props);
 
+    this.state = {
+      incomingMessage: {
+        message: "",
+        userFrom: NOOP_USERNAME,
+        userFromID: NOOP_ID
+      }
```

Comment:
Initialize the state. This will never be interpreted as an actual state as `ChatHistory` will only make `Message` components on `componentWillReceiveProps` lifecycle events.